server/controllers/itemsController.js

```diff
@@ -1,16 +1,32 @@
 const AppError = require('../utils/AppError');
+const db = require ('../models.js');
+
 
 const itemsController = {};
 
 itemsController.getAllItems = async (req, res, next) => {
-  return next(new AppError(new Error('Not implemented'), 'itemsController', 'getItems', 500));
+  try {
+    // eslint-disable-next-line semi
+    const text = 'SELECT * FROM public."Items" ORDER BY name ASC'
```

Comment:
In the Chapters controller the table is referenced as public.chapters. The use of quotes here is required because the Items table has been defined with a capital "I"?

Reply:
I actually can now that I am in PgAdmin! It defaulted to that in dbdesigner for some reason?